core/src/main/java/org/infinispan/distribution/ch/ConsistentHash.java

```diff
@@ -112,31 +112,10 @@ default boolean isReplicated() {
     * Writes this {@link ConsistentHash} to the specified scoped persistent state.
     *
     * @param state         The state to which this {@link ConsistentHash} will be written.
-    * @param addressMapper The mapper {@link Function} to convert the {@link Address} to the unique string used to
-    *                      persist the address within the state.
+    * @param addressMapper The mapper {@link Function} to convert the {@link Address} to the {@link PersistentUUID} used
+    *                      to persist the address within the state.
     */
-   default void toScopedState(ScopedPersistentState state, Function<Address, String> addressMapper) {
```

Comment:
This interface is part of the exposed API. Should we be deprecating these methods and marking for removal instead of removing now?